packages/netifyd/Makefile

```
@@ -16,9 +16,10 @@ PKG_INSTALL:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.com/netify.ai/public/netify-agent.git
-PKG_SOURCE_DATE:=2022-11-07
-PKG_SOURCE_VERSION:=189a993a831a3eb74951fc19ea54e6144263c09e
-PKG_MIRROR_HASH:=9762c98053fdc7f6f40560b6af9638cbe380dfc84201ffc91ec1e1005323b06c
+PKG_SOURCE_DATE:=2023-06-15
+PKG_SOURCE_VERSION:=v4.4.3
+#PKG_SOURCE_VERSION:=411df94c131e034d0f30ff6773e6871b23640f01
+PKG_MIRROR_HASH:=whatever
```

I'm not sure switching to the tagged release since upstream is not really accurate about versioning.

Still, why version 4.4.3 while upstream is 4.4.1 https://gitlab.com/netify.ai/private/nethesis/netify-flow-actions/-/commit/81ea8c4127238f230f919b2718ead07862d0a233?
```suggestion
PKG_SOURCE_DATE:=2023-06-15
PKG_SOURCE_VERSION:=v4.4.1
PKG_MIRROR_HASH:=whatever
```